modules/ROOT/pages/tutorial/tutorial-clustering-docker.adoc

```diff
@@ -32,9 +32,58 @@ For more information, see the https://docs.docker.com/compose/install/[Install D
 
 *Procedure*
 
-. Create a configuration file `neo4j.conf` which will be shared across cluster members and make it readable and writable for the user (eg., `chmod 640 neo4j.conf`)
+. Create a configuration file _neo4j.conf_ which will be shared across cluster members and make it readable and writable for the user (eg., `chmod 640 neo4j.conf`)
 +
-====
+[.tabbed-example]
+=====
+[role=include-with-discovery-service-v2 label--new-5.23]
+======
+[source,shell,subs="attributes+, +macros"]
+----
+# Setting that specifies how much memory Neo4j is allowed to use for the page cache.
+server.memory.pagecache.size=100M
+
+# Setting that specifies the initial JVM heap size.
+server.memory.heap.initial_size=100M
+
+# The behavior of the discovery service is determined by the parameters `dbms.cluster.discovery.resolver_type`, `dbms.cluster.discovery.v2.endpoints`, and `dbms.cluster.discovery.version`.
+# The DNS strategy fetches the IP addresses of the cluster members using the DNS A records.
+dbms.cluster.discovery.resolver_type=DNS
+
+# The value of `dbms.cluster.discovery.version` must be set to `V2_ONLY` if you want to use the discovery service v2.
+# The discovery service v2 utilizes the port `6000`.
+
+# The value of `dbms.cluster.discovery.V2.endpoints` should be set to a single domain name and the port of the discovery service.
+# The domain name returns an A record for every server in the cluster when a DNS lookup is performed.
+# Each A record returned by DNS should contain the IP address of the server in the cluster.
+# The configured server uses all the IP addresses from the A records to join or form a cluster.
+# The discovery port must be the same on all servers when using this configuration.
+dbms.cluster.discovery.V2.endpoints=neo4j-network:6000
+
+# Address (the public hostname/IP address of the machine)
+# and port setting that specifies where this instance advertises for discovery protocol messages from other members of the cluster.
+server.cluster.advertised_address=$(hostname -i)
+
+# Address (the public hostname/IP address of the machine)
+# and port setting that specifies where this instance advertises for Raft messages within the cluster.
+server.cluster.raft.advertised_address=$(hostname)
+
+ # Address (the public hostname/IP address of the machine)
+ # and port setting that specifies where this instance advertises for requests for transactions in the transaction-shipping catch-up protocol.
+server.cluster.advertised_address=$(hostname)
+
+# Enable server-side routing
+dbms.routing.enabled=true
+
+# Use server-side routing for neo4j:// protocol connections.
+dbms.routing.default_router=SERVER
+
+# The advertised address for the intra-cluster routing connector.
+server.routing.advertised_address=$(hostname)
+----
```

Comment:
```suggestion
# Automatically enable servers, rather than needing to explicitly do so for Free servers                                                                                                                              
initial.dbms.automatically_enable_free_servers=true
----
```